scripts/app.js

```diff
@@ -103,84 +103,60 @@ function InitializeGrid() {
 //        ws.send(JSON.stringify(request));
 //    }
 
-    $("#notification").kendoNotification({
-        width: "100%",
-        position: {
-            top: 0,
-            left: 0
-        }
-    });
+    // $("#notification").kendoNotification({
+    //     width: "100%",
+    //     position: {
+    //         top: 0,
+    //         left: 0
+    //     }
+    // });
 
     $("#grid").kendoGrid({
         height: 550,
-        autoBind: false,
+        //autoBind: false,
         editable: true,
         sortable: true,
         columns: [
-                    { field: "UnitPrice" },
-                    { field: "ProductName" },
-                    { command: "destroy", width: 150 }
+                    { field: "connections", title: "Connections" },
```

Comment:
the "field" property here should be named the same as the property of the object coming from the server. Then you can add "title" so you can have a better header for your column